tools/mm-macros/Makefile

```
@@ -8,26 +8,20 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mm-macros
-PKG_VERSION:=1.0.0
+PKG_VERSION:=1.0.3
 
 PKG_SOURCE_URL:=@GNOME/mm-common/1.0
 PKG_SOURCE:=mm-common-$(PKG_VERSION).tar.xz
-PKG_HASH:=b97d9b041e5952486cab620b44ab09f6013a478f43b6699ae899b8a4da189cd4
+PKG_HASH:=e81596625899aacf1d0bf27ccc2fcc7f373405ec48735ca1c7273c0fbcdc1ef5
 
 HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/mm-common-$(PKG_VERSION)
 
-HOST_FIXUP:=autoreconf
-
 include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/meson.mk
 
 define Host/Install
```

```suggestion
define Host/Install
 	$(call Host/Install/Meson)
```

Isn't that needed too?

---

Good question. I don't think it is.

So the Install section just installs the m4 macros. Calling $(call Host/Install/Meson) adds that, some doc and manual stuff, and a bin. I think this is intentional not to install everything.